api/server.py

```diff
@@ -192,8 +192,8 @@ def profile_dataset(
 
 
 @app.post("/profile_model/{model_id}")
-def profile_model(model_id: str, paper_artifact_id: str, redis=Depends(get_redis)) -> ExtractionJob:
-    """Profile model with MIT's profiling service. This takes in a paper and code artifact
+def profile_model(model_id: str, paper_document_id: str, redis=Depends(get_redis)) -> ExtractionJob:
```

Comment:
should this just be `document_id`? I think the `paper_` prepend was just to differentiate between the paper and code artifacts